_user/hellocpp/main.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Hello World Cpp
+ *
+ * Example of user application
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Dawid Szpejna
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <iostream>
+
+
+int main(void)
+{
+	std::cout << "Hello World Cpp!" << std::endl;
```

Comment:
In my opinion it would be better, (eventually if you want to use cpp phrase there I'd rather print sth like `"Hello CppWorld!"`)

```suggestion
	std::cout << "Hello World!" << std::endl;
```

Reply:
How about `Hello World++!`? :smile:

Reply:
@damianloew I corrected the string